pkgs/tools/security/pinentry/default.nix

```diff
@@ -35,29 +36,29 @@ let
     gnome3 = { bin = "gnome3"; flag = "gnome3"; buildInputs = [ gcr ]; nativeBuildInputs = [ wrapGAppsHook ]; };
     qt = { bin = "qt"; flag = "qt"; buildInputs = [ qtbase ]; nativeBuildInputs = [ wrapQtAppsHook ]; };
     emacs = { bin = "emacs"; flag = "emacs"; buildInputs = []; };
+    efl = { bin = "efl"; flag = "efl"; buildInputs = [ pkgs.enlightenment.efl ]; };
   };
 
 in
 
 pinentryMkDerivation rec {
   pname = "pinentry";
-  version = "1.1.0";
+  version = "ae584040d14a4d7d6eed8be77844126130ef0118";
```

Comment:
According to https://nixos.org/nixpkgs/manual/#sec-package-naming, the `version` should be `unstable-yyyy-mm-dd`, when using git snapshots.